### PR TITLE
Add support for tags

### PR DESCRIPTION
**Pull request by Universal-Omega**
## Summary by CodeRabbit

- **New Features**
  - Introduced a new "wiki tags" feature, allowing users to tag wikis with specific labels.
  - Added a dynamic tag selection interface for easier tagging.

- **Enhancements**
  - Updated forms to include a field for wiki tags, enhancing the user experience by allowing more detailed categorization.

- **Localization**
  - Added new localization strings for "wiki tags" in English.

- **Database**
  - Added a new column for storing wiki tags in the database schema.

### CODE REVIEW

**Comment by Universal-Omega** [comment]
@coderabbitai review

File: includes/RequestWiki/SpecialRequestWiki.php [hunk 1]
```diff
@@ -95,6 +103,15 @@ protected function getFormFields() {
 			];
 		}
 
+		if ( $this->config->get( 'CreateWikiUseWikiTags' ) ) {
+			$formDescriptor['wikitags'] = [
+				'type' => 'text',
+				'label-message' => 'createwiki-label-wiki-tags',
+				'cssclass' => 'createwiki-wikitags',
+				'disabled' => true,
```
Comment (OAuthority) [inline includes/RequestWiki/SpecialRequestWiki.php:111]: This should be `readonly => true`, no? Or else it won't be sent with the form?

Comment (Universal-Omega) [reply includes/RequestWiki/SpecialRequestWiki.php:111]: You may be right. This still has a lot of work like also passing to cw_requests also

File: modules/ext.createwiki.oouiform.wikiTags.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+( function () {
```
Comment (BlankEclair) [inline modules/ext.createwiki.oouiform.wikiTags.js:1]: Do we really need two IIFEs?

File: includes/RequestWiki/SpecialRequestWiki.php [hunk 3]
```diff
@@ -100,6 +105,15 @@ protected function getFormFields(): array {
 			];
 		}
 
+		if ( $this->getConfig()->get( ConfigNames::AvailableTags ) {
+			$formDescriptor['wikitags'] = [
```
Comment (BlankEclair) [inline includes/RequestWiki/SpecialRequestWiki.php:109]: What happens when someone requests more than five tags?

**Comment by Universal-Omega** [comment]
I will redo this a different way later.

**Pull request closed without merging**